app/views/councilTaxUpload.scala.html

```diff
@@ -61,7 +63,13 @@ <h1 class="govuk-label-wrapper">
                     id = "file",
                     name = "file",
                     classes = "govuk-form-group upload",
-                    attributes = Map("accept" -> "application/xml", "required"  -> "required")
+                    errorMessage = fileUploadForm.errors.asTextErrorMessageForField("file"),
+                    attributes =
+                        Map(
+                            "accept" -> "application/xml",
+                            "required"  -> "required",
+                            "oninvalid" ->  ("window.location.href='" + routes.CouncilTaxUploadController.onPageLoad(true).url + "#file';")
```

Comment:
`oninvalid` allow to override the default behaviour of the error handling on the form (in this instance file empty caused by the "required" attributes)... It's not @rhubner exactly what we've discussed - but it seems to do the trick...

Reply:
We should test in `IE11`. I will expect this will be the main browser what BA is using. Maybe Andrew will have more accurate data.